features/TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# [ Feature Title ]
+
+## Problem Summary
+
+> Talk about current goals of the system and how they aren't being met. What outcomes are we not delivering? This can also include an explicit request for improvement that doesn't dictate a specific or concrete solution.
+
+### Assumptions
+
+> This is where you talk about what you assume to be true. This could include assumptions around what users are doing, errors, gaps, etc., based on anecdotes, opinions, testing, or data.
+
+****
+
+## Outcomes
+
+> What are the outcomes you want to achieve? What is the success criteria?
+
+### Metrics
+
+> How are you going to measure the outcome / success?
+
+> **Please provide sample artifact graphs here.**
+
+## Hypothesis
+
+> A high level hypothesis of how the feature you're going propose is going help us achieve the outcomes listed above. I recommend this be a sentence of the form:
+
+> We believe that
+(doing this/building this/creating this experience)
+for (these people)
+will achieve (this outcome).
+We will know this is true when we see this
+(qualitative feedback/quantitative feedback/KPI change).
+
+****
+
+## Detailed design
+
+> This is the bulk of the RFC. Explain the design in enough detail for somebody familiar
+with the language to understand.
+This should get into specifics and corner-cases, and include examples of how the feature is used.
+
+
+### Unresolved questions (optional)
```

Comment:
I like to call out both unresolved questions and risks. Thinking through risks and what can derail the project both minimizes surprises and allows contingencies to be developed.